src/Conversion/KrnlToLLVM/ConvertKrnlToLLVM.cpp

```diff
@@ -410,6 +410,11 @@ void ConvertKrnlToLLVMPass::runOnOperation() {
   MLIRContext *ctx = &getContext();
   const auto &dataLayoutAnalysis = getAnalysis<DataLayoutAnalysis>();
   LowerToLLVMOptions options(ctx, dataLayoutAnalysis.getAtOrAbove(module));
+
+  // There are many places where we still rely on non-opaque pointers. Disable
+  // opaque-pointers until we migrated the affected code parts
+  options.useOpaquePointers = false;
```

Comment:
Thanks for pointing it out. I can't find any references to it being used though, so it might just be an unused instance.

In any case, leaving opaque pointers enabled there doesn't affect any tests. All tests are passing locally for me without this change.

Reply:
Thanks!